frontend/lint/tests/no-unconditional-metabase-links-render.unit.spec.js

```diff
@@ -0,0 +1,185 @@
+import { RuleTester } from "eslint";
```

Comment:
All cases are performed according to the ESLint rule spec
https://github.com/metabase/metabase/blob/f992c45dcae991d90704c249c7c3fec49d9ab0d7/frontend/lint/eslint-rules/no-unconditional-metabase-links-render.js#L5-L15